### ISSUE TITLE
Support for TSLint 5.x

### ISSUE THREAD

**Issue by darxriggs** [nicolas-schmitt/gulp-tslint-jenkins-reporter#16]
With TSLint 5 `failureCount` was renamed to `errorCount`.

gulp-tslint 8.0.0 already supports this (see [CHANGELOG.md](https://github.com/panuhorsmalahti/gulp-tslint/blob/master/CHANGELOG.md))

**Comment by nicolas-schmitt** [comment]
Hey, thanks for the heads up 😸 
However, the error count isn't part of the checkstyle format (as far as i know) and I'm therefore not using it in the formatter. 

I know i'm using a "failureCount" property in my unit tests but solely as a mock data and not compared directly to what tslint provides. Thus not in dire need of fixing.

Did you have any specific fix/feature in mind ?

**Comment by darxriggs** [comment]
The peer dependencies in _package.json_  don't list gulp-tslint 8.0.0 which is required for tslint 5.0.0.

```
"peerDependencies": {
  "gulp-tslint": "^5.0.0 || ^6.1.3 || ^7.0.1"
},
```

**Comment by nicolas-schmitt** [comment]
oh ok then this has been fixed in the latest version : 1.1.0
let me know if there's anything else to fix :)
